common/containers/Tabs/ScheduleTransaction/components/Fields/ScheduleTimestamp/ScheduleTimestampFieldFactory/ScheduleTimestampInputFactory.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { Query } from 'components/renderCbs';
+import {
+  getCurrentScheduleTimestamp,
+  isValidCurrentScheduleTimestamp,
+  ICurrentScheduleTimestamp,
+  fiveMinFromNow
+} from 'selectors/transaction';
+import { connect } from 'react-redux';
+import { AppState } from 'reducers';
+import { CallbackProps } from './ScheduleTimestampFieldFactory';
+import { getResolvingDomain } from 'selectors/ens';
+import Pikaday from 'pikaday-time';
+import moment from 'moment';
+import { EAC_SCHEDULING_CONFIG } from 'libs/scheduling';
+
+import { setCurrentScheduleTimestamp, TSetCurrentScheduleTimestamp } from 'actions/transaction';
+
+interface DispatchProps {
+  setCurrentScheduleTimestamp: TSetCurrentScheduleTimestamp;
+}
+
+interface StateProps {
+  currentScheduleTimestamp: ICurrentScheduleTimestamp;
+  isValid: boolean;
+  isResolving: boolean;
+}
+
+interface OwnProps {
+  onChange(ev: React.FormEvent<HTMLInputElement>): void;
+  withProps(props: CallbackProps): React.ReactElement<any> | null;
+}
+
+type Props = DispatchProps & OwnProps & StateProps;
+
+class ScheduleTimestampInputFactoryClass extends Component<Props> {
+  public componentDidMount() {
+    const datetimepickerElement = document.getElementById('datepicker');
+    const { currentScheduleTimestamp, onChange } = this.props;
+
+    const picker = new Pikaday({
```

Comment:
This won't play nice if the component dismounts. Could we use https://github.com/thomasboyt/react-pikaday instead? I also noticed that this doesn't follow the component as you scroll.